Gibe.Umbraco.Blog/Composing/IndexEventsComponent.cs

```diff
@@ -133,40 +133,67 @@ private void ExternalIndexTransformingIndexValues(object sender, IndexingItemEve
 
 		private Dictionary<string, IEnumerable<object>> AddAuthorFields(IndexingItemEventArgs e)
 		{
-			var authorId = e.ValueSet.GetSingleValue<int?>(ExamineFields.PostAuthor);
+			var authorIdString = e.ValueSet.GetSingleValue(ExamineFields.PostAuthor);
+			var authorName = string.Empty;
+			if(int.TryParse(authorIdString, out var authorId))
+			{
+				authorName = GetUserName(authorId);
+			}
+			else if (UdiParser.TryParse(authorIdString, out var authorUid))
```

Comment:
Nitpick/readability: Maybe to make this clearer for future code-readers, we could put this into a local method?
```suggestion
			else if (IsContentNodeUdi(authorIdString, out var authorUdi))
```